daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/_index.md

```diff
@@ -2,9 +2,447 @@
 type: docs
 title: "Getting started with the Dapr actors .NET SDK"
 linkTitle: "Actors"
-weight: 40000
-description: How to get up and running with the Dapr .NET SDK
+weight: 100000
+description: Try out .NET virtual actors with this example
```

Comment:
Note: this is about 75% existing content and 25% new descriptions of old content. 

I don't mind making small improvements here, but ultimately this should grow up into a real tutorial someday that actually explains the concepts and how to use them.

Reply:
Yeah I agree. There's also the issue of this content existing in the examples in the repo. We should come up with a long term plan on this, I expect it will align with what we discussed on pulling in code from the repos into the docs